controllers/suite_util_test.go

```diff
@@ -296,6 +296,27 @@ func waitImageRepositoryFinalizerOnImageRepository(imageRepositoryKey types.Name
 	}, timeout, interval).Should(BeTrue())
 }
 
+// func waitImageRepositoryCredentialGone(imageRepositoryKey types.NamespacedName, credentialName string) {
+func waitImageRepositoryCredentialGone(imageRepositoryKey types.NamespacedName, operationName string) {
```

Comment:
Name is confusing. It seems that it waits until the secrets with credentials gone...

Reply:
it waits until imageRepository.Spec.Credentials sections are gone

hence the name : ImageRepositoryCredential

suggest better one if you have idea

Reply:
`waitImageRepositoryCredentialSectionGone` at least? Or maybe `waitImageRepositoryCredentialSectionRequestGone`?

Reply:
ok renamed